addons/website_slides/static/src/js/activity.js

```diff
@@ -1,45 +1,47 @@
 odoo.define('website_slides.Activity', function (require) {
-"use strict";
+    "use strict";
 
-var field_registry = require('web.field_registry');
+    const components = {
+        Activity: require('mail/static/src/components/activity/activity.js'),
+    };
 
-require('mail.Activity');
+    const { patch } = require('web.utils');
 
-var KanbanActivity = field_registry.get('kanban_activity');
```

Comment:
I don't think we should remove the override of KanbanActivity in stable, old activity code is still used in some places.

This PR should just introduce the new overrides to be safe.